### ISSUE TITLE
Syntaxic Coloration

### ISSUE THREAD

**Issue by Drasenix** [GitFiend/Support#165]
Hello,

For better readability, It could be interresting to have syntaxic coloration for the code inside commited files.

**Comment by GitFiend** [comment]
Yeah, agree. I plan on doing this once I find a nice solution for it.
